### ISSUE TITLE
Select active boorus on list

### ISSUE THREAD

**Issue by Cerberuschan** [flexbooru/flexbooru#97]
If possible make a way to select the active boorus on non paid version, I know everyone needs cash but I don't know if I can put money on a app that is another version of a dead one man... I used to use moebooru a lot, and look what happened.

**Comment by DonaldTsang** [comment]
Moebooru is made by the same people, but yeah.

**Comment by Cerberuschan** [comment]
@DonaldTsang  that's exactly why I said it, how can one assures me that they will not do so like they did with moebooru? Just make a new one and abandon flex because it's old and it's easier to make a new one? 
They are locking pretty basic stuff behind a pay wall. For now I'll stay with anime boxes, gives everything for free. Besides I wouldn't mind buying the app, but it's kind expensive for a booru reader, it's 20 bucks around here!

**Comment by DonaldTsang** [comment]
@Cerberuschan well with the source code open, it would be much better to have a working group to improve the code at any time to beak close sourced AnimeBoxes

**Comment by Cerberuschan** [comment]
@DonaldTsang then how about someone forked the app and make a useful one? Without all the paywall crap. Anime boxes might be closed source but at least it doesn't lock everything behind paywall and don't have those annoying advertisements.

**Comment by DonaldTsang** [comment]
@Cerberuschan you might want to check the Discord where someone rewrite the app for easier tooling. https://discord.gg/zxAX5Jh `hunterx9#1128` is the name you are looking for.

**Comment by manishxshrestha** [comment]
@Cerberuschan  hey if you got the cash then please support the original dev, i am rewriting the app since original dev has shown less interest, join in if u want to be a tester

**Comment by DonaldTsang** [comment]
@hunterx9 please fork the code and put it on GitHub if you can, you might want to learn Git or use VSCode to help you use Git without learning the command line.
- https://code.visualstudio.com/blogs/2018/09/10/introducing-github-pullrequests
- https://dev.to/phillipdodd/how-to-set-up-github-projects-in-a-vs-code-workspace-2j27
- https://medium.com/@brygrill/version-control-basics-with-github-and-vs-code-1c1906cadd33

**Comment by hunterx9** [comment]
i have it on dev branch in my forked flexbooru
https://github.com/hunterx9/flexbooru

Check it out and any supporters for building the app is greatly appreciated. dm me on flexbooru discord

**Comment by Cerberuschan** [comment]
Man thx, but pardon my ignorance, how do I download and install your fork? 
I'm on Android 8.1. 
thx in advance.

**Comment by hunterx9** [comment]
Install android studio and git and then change branch ti deva nd run in your phone, if thats difficult for you join my discord server i have the apk

**Comment by hunterx9** [comment]
https://discord.gg/aXb7JWe

**Comment by DonaldTsang** [comment]
@hunterx9 do you know how to do releases in GitHub? For example https://github.com/flexbooru/flexbooru/releases ? (Of course label your releases slightly differently to prevent clashing and allow for easier pull requests)

**Comment by Cerberuschan** [comment]
Thanks man for the response, but that's too complicated for me. I think I'll wait you release here on git the way Donald just said.

Thanks in advance man. Sorry if I'm giving you too much trouble.

**Comment by hunterx9** [comment]
Join my discord server for now there is download link for app, i wont be on my computer for a few days

**Comment by Cerberuschan** [comment]
You guys, your app does not work with shimmie 2 and gelbooru 0.1.11? That narrowed my options of boorus. Can we get support for those too?

**Comment by DonaldTsang** [comment]
@Cerberuschan yes, we will add support for that as well, could you tell me what sites uses Shimme2? I know Gelbooru 0.1.11 is for booru.org right?

**Comment by Cerberuschan** [comment]
Yes of course, but first I must say that it's all NSFW sites.
That said, the ones I use are 
- gelbooru 0.1.11 I use on lm.booru.org
- shimmie 2 I use on rule34.paheal.net and rule34hentai.net
Visit them at your on risk, make sure no judging eyes a preying on you.
